api/src/main/java/org/hawkular/apm/api/services/StoreException.java

```diff
@@ -14,25 +14,44 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.hawkular.apm.server.elasticsearch;
+package org.hawkular.apm.api.services;
 
 /**
- * This class is an exception representing Elasticsearch failures.
+ * This exception indicates a failure to store information.
  *
  * @author gbrown
  */
-public class ElasticsearchFailures extends Exception {
+public class StoreException extends Exception {
```

Comment:
Nitpick: Given this is a checked exception and is used in IO-related scenarios, maybe this could be replaced by the `IOException` to save 1 class definition. On the other hand the name is maybe more descriptive.